lib/peep.rb

```diff
@@ -0,0 +1,67 @@
+require 'pg'
+
+class Peep
+
+  attr_reader :id, :peep, :time, :account
+
+  def initialize(id:, peep:, time:, account:)
+    @id = id
+    @peep = peep
+    @time = time
+    @account = account
+  end
+
+
+  def self.all
+      result = DatabaseConnection.query("SELECT * FROM peeps ORDER BY id desc;")
+      result.map do |peep|
+      Peep.new(id: peep['id'], peep: peep['peep'], time: peep['time'], account: peep['account'])
+      end
+    end
+
+    def self.add(peep:, account:)
+      if ENV['ENVIRONMENT'] == 'test'
```

Comment:
I think because of your database connection class file and requiring the set up file at the top of peep.rb, you don't need the set up in the methods. Like you've done in .all you would only need DatabaseConnection.query?